Makefile

```diff
@@ -42,6 +42,11 @@ test-go:
 	go test ./... -race -timeout 30s
 
 
+.PHONY: test-integration
+test-integration:
+	go test ./integration-tests -timeout 185s --integration
```

Comment:
185s? How long do we expect them to take?

Reply:
Ah yep, ran into that with webpack before. Really surprising it's not more optimized.